data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml

```diff
@@ -263,6 +263,9 @@
         </createTable>
         <addPrimaryKey columnNames="id" constraintName="virtual_node_operation_pk" tableName="virtual_node_operation"
                        schemaName="${schema.name}"/>
+        <createIndex indexName="virtual_node_operation_request_id" tableName="virtual_node_operation" schemaName="${schema.name}">
+            <column name="request_id"/>
+        </createIndex>
```

Comment:
We'll be querying the virtual node operation table using the `request_id`, not the primary key `id`, FYI @vkolomeyko on this index I suggested adding.